frontend/src/metabase/visualizations/components/ChartSettings/ChartSettingsFooter.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from "ttag";
+
+import Button from "metabase/core/components/Button";
+
+import { ChartSettingsFooterRoot } from "./ChartSettings.styled";
+
+export const ChartSettingsFooter = ({
+  onDone,
+  onCancel,
+  onReset,
+}: {
```

Comment:
nit: conventionally it would be nice to define an interface above

Reply:
yep, definitely. good call